setup.py

```diff
@@ -16,7 +16,7 @@
 cpp_extension = CppExtension(
     name="norse_op",
     sources=["norse/csrc/op.cpp", "norse/csrc/super.cpp"],
-    extra_compile_args=["-O3"],
+    extra_compile_args=["-O3", "/std:c++17"],
```

Comment:
this wouldn't work on Linux.

Reply:
OK. I've added one more change to solve this. I hope you like it.